demo/settings/prod.py

```diff
@@ -46,6 +46,14 @@
 SLACK_SIGNING_SECRET = get_env_var("SLACK_SIGNING_SECRET")
 INCIDENT_CHANNEL_NAME = get_env_var("INCIDENT_CHANNEL_NAME")
 INCIDENT_BOT_NAME = get_env_var("INCIDENT_BOT_NAME")
-
+SECRET_KEY = get_env_var("SECRET_KEY")
 INCIDENT_BOT_ID = os.getenv("INCIDENT_BOT_ID") or SLACK_CLIENT.get_user_id(INCIDENT_BOT_NAME)
 INCIDENT_CHANNEL_ID = os.getenv("INCIDENT_CHANNEL_ID") or SLACK_CLIENT.get_channel_id(INCIDENT_CHANNEL_NAME)
+STATUS_PAGE_RUNBOOK_URL = get_env_var("STATUS_PAGE_RUNBOOK_URL")
+RUNBOOKS_URL = get_env_var("RUNBOOKS_URL")
```

Comment:
I think we spoke about this offline, but now we have our own code base and are pulling Response as a package now, can we switch to using playbooks rather than runbooks?

Can we also only add configuration when we're adding features? For example, can we remove PagerDuty configuration till we're ready to integrate this as the configuration might confuse people if not used.

Reply:
Yep - removing the defaults.